wasi/src/main/java/com/dylibso/chicory/wasi/WasiFstFlags.java

```diff
@@ -0,0 +1,14 @@
+package com.dylibso.chicory.wasi;
+
+final class WasiFstFlags {
+    private WasiFstFlags() {}
+
+    public static final int ATIM = bit(0);
```

Comment:
can you add a comment with a link to lookup those definitions?

Reply:
I added another commit with doc links for all the classes

Reply:
Can you, please, use permalinks?

Linking to `main/blob` tend to go out of date pretty fast in my experience.

Reply:
I can do that if you feel strongly, but I think it's better to link to the current documentation and update the link if it breaks, rather than linking to an outdated version (without noticing that it's old).

Given that preview1 is basically stable, I'm not sure it matters either way.

Reply:
I do prefer, almost always, permalinks over moving tags:

- they don't get outdated 
- they show the link the author used to do the implementation, including errors etc.

That said, the comment is close to a nitpick, not going to block this great effort for it. Thanks 👍

Reply:
For code links, especially with line numbers, I complete agree.

Whenever I see a documentation link to a commit, my first thought is "What's this random commit? Let me go find the latest."

So that's my aversion to doing it. But I do see your point view. Ideally, they would publish the spec to a website with stable URLs.

Reply:
following up only to challenge my stand point and because I enjoy this discussion 😊:
when you get the latest you will have a hard time mixing and matching the date on the git commit and the actual status of the repo at that point in time, to understand what the author followed.

I usually find the inverse process much easier to follow (e.g. forwarding changes to see what changed) as it gives you a stable and fixed starting point to reason about.

That said, yes, stable URLs will be best...

Reply:
Alright, you convinced me. I'll update it to the latest release tag. Also, the README was pointing to `main` and the link was broken 😆

Reply:
hahah :+1: